tools/torchci/download_logs.py

```diff
@@ -47,15 +60,47 @@ def download_logs_to_dir(commit):
     return folder
 
 
+def download_artifacts_from_sha(commit):
+    s3 = get_s3()
+    bucket = s3.Bucket("gha-artifacts")
+    folder = REPO_ROOT / "_logs" / "artifacts" / commit
+    workflow_ids = query_rockset(
+        f"select id from workflow_run where head_sha = '{commit}'"
+    )
+
+    zipped_path = folder / "zipped"
+    os.makedirs(zipped_path, exist_ok=True)
+
+    for row in workflow_ids:
+        workflow_id = row["id"]
+        for obj in bucket.objects.filter(Prefix=f"pytorch/pytorch/{workflow_id}"):
```

Comment:
Making this repo name configurable is a good idea, so it can be used elsewhere too, i.e. `pytorch/executorch`